src/expire.c

```diff
@@ -489,11 +489,11 @@ int checkAlreadyExpired(long long when) {
     return (when <= mstime() && !server.loading && !server.masterhost);
 }
 
-#define FLAG_NO_FLAGS 0
-#define FLAG_NX (1<<0)
-#define FLAG_XX (1<<1)
-#define FLAG_GT (1<<2)
-#define FLAG_LT (1<<3)
+#define EXPIRE_NO_EXPIRES 0
```

Comment:
i think that's a bad name for that flag.
it designates that there are no flags, not that there is on expire.
got also, being 0, it's not really a flag, and i think we actually don't need it.
just use hard coded 0 instead (for an integer named "flags" it's logical to just init to 0)